packages/discovery-react-components/src/components/SearchFacets/utils/searchFilterTransform.tsx

```diff
@@ -67,36 +69,40 @@ export class SearchFilterTransform {
 
   static toString(facets: SearchFilterFacets): string {
     const fieldFilters = this.fieldsToString(facets.filterFields);
-    const dynamicFilters = this.quoteSelectedFacets(facets.filterDynamic, 'text').join(',');
+    const dynamicFilters = this.escapeSelectedFacets(facets.filterDynamic, 'text').join(',');
     return [fieldFilters, dynamicFilters].filter(Boolean).join(',');
   }
 
-  private static unquoteString(quotedString: string): string {
-    return quotedString.replace(/^"(.+)"$/, '$1').replace(/\\"/, '"');
+  private static unescapeString(escapedString: string): string {
+    const unescapedString = escapedString
+      .replace(/^\((.+)\)$/, '$1')
+      .replace(RegExp(SearchFilterTransform.ESCAPED_CHAR), '$1');
+    return unescapedString;
   }
 
   static fieldsToString(facets: InternalQueryTermAggregation[]): string {
     const filterStrings: string[] = [];
     facets.forEach(facet => {
       const field = get(facet, 'field', '');
       const results = get(facet, 'results', []);
-      const keys = this.quoteSelectedFacets(results, 'key');
+      const keys = this.escapeSelectedFacets(results, 'key');
       if (keys.length) {
-        filterStrings.push(keys.map(key => `${escapeFieldName(field)}::${key}`).join('|'));
+        filterStrings.push(keys.map(key => `${escapeFieldName(field)}:${key}`).join('|'));
```

Comment:
Add a comment here, about the use of `:` and what it means.

Reply:
Actually, make the comment on this function, specifying that it is creating a query string with ~~exact match~~ whatever we're doing now, etc.